appmap/test/test_django.py

```diff
@@ -1,23 +1,55 @@
+# flake8: noqa: E402
+# pylint: disable=unused-import, redefined-outer-name
+
 import pytest
 from appmap._implementation.recording import Recorder
 
 pytest.importorskip('django')
 
+import django.conf
+import django.db
+import django.test
+import appmap.django  # noqa: F401
+
+django.conf.settings.configure(
+    DATABASES={'default': {'ENGINE': 'django.db.backends.sqlite3', 'NAME': ':memory:'}},
+    ROOT_URLCONF=()
+)
+
+
+@pytest.fixture
+def recorder():
+    rec = Recorder()
+    rec.events().clear()
+    rec.enabled = True
+    yield rec
+    rec.enabled = False
+    rec.events().clear()
 
-def test_sql_capture():
-    # pylint: disable=unused-import
-    import django.conf
-    import django.db
-    import appmap.django  # noqa: F401
 
-    recorder = Recorder()
-    recorder.events().clear()
-    recorder.enabled = True
+@pytest.fixture
+def events(recorder):
+    return recorder.events
```

Comment:
Maybe this should be `yield events` instead?

Reply:
Why? There's no cleanup that needs to be done.

Reply:
I think it's confusing the way it is, and would be more idiomatic if it used `yield`. The fact that `events` isn't a property, when it almost certainly should be, contributes to the confusion.

So, if you added `@property` to `Recorder.events`, you could say

```
@pytest.fixture
def events(recorder):
    yield recorder.events
```
and users of the fixture could just reference `events` with having to call it as a function.

Reply:
I agree that maybe it should be a property, however I fail to see how using yield would make any difference or how it would be more idiomatic. I suspect one of us (probably me) is misunderstanding how python coroutines (or whatever it's called that uses yield) and pytest fixtures work. I just followed pytest fixture documentation, which uses return for fixtures normally unless they need cleanup after usage.

Reply:
Anyway, I've reworked the fixture a little bit, hopefully you'll like it better ;)

Reply:
I do like it better, thanks. :)

I guess my issue wasn't really `yield` versus `return`, it was that the output from the fixture was a function. Now that you've eliminated that, it looks good to me.